test/util/edu-1990e-helpers.js

```diff
@@ -0,0 +1,22 @@
+const mock = require('./mock-helpers');
+
+// Create API routes
+function initApplicationSubmitMock() {
```

Comment:
Can this also be shared with the other education forms? Looks like they're exactly the same except for the end of the endpoint, which corresponds with the form number.